libpromises/evalfunction.c

```diff
@@ -736,16 +736,35 @@ static FnCallResult FnCallHostsSeen(ARG_UNUSED EvalContext *ctx, ARG_UNUSED cons
 }
 
 /*********************************************************************/
+enum {
+    HOSTSWITHCLASS,
+    HOSTSWITHGROUP
```

Comment:
Easier to read in my opinion. But feel free to dismiss if you don't agree
```suggestion
    HOSTS_WITH_CLASS,
    HOSTS_WITH_GROUP
```